ambry-clustermap/src/main/java/com/github/ambry/clustermap/HelixParticipant.java

```diff
@@ -439,7 +441,7 @@ private void awaitDisablingPartition() throws InterruptedException {
    * @param callback a callback which will be invoked when the aggregation report has been generated successfully.
    */
   private void registerHealthReportTasks(StateMachineEngine engine, List<AmbryHealthReport> healthReports,
-      Callback<StatsSnapshot> callback) {
+      AccountStatsStore accountStatsStore, Callback<StatsSnapshot> callback) {
```

Comment:
same here. (Also, we should consider removing the callback, this can be done in future PR)